OmniBLE/PumpManager/PendingCommand.swift

```diff
@@ -8,14 +8,15 @@
 
 import Foundation
 import LoopKit
+import OmniKit
```

Comment:
Can this import be avoided? I am hoping that eventually all the code in OmniBLE/OmnipodCommon can be made into a separate Framework that can be shared between OmniKit and OmniBLE. With some more work, it might be possible to share even more code between OmniKit and OmniBLE.

Reply:
Yep